### ISSUE TITLE
Inconsistent use of Loo v Toilet on the 'Add Toilet' screen

### ISSUE THREAD

**Issue by pw3032** [neontribe/gbptm#258]
Most of the site uses the term 'toilet' this screen uses the word 'loo' in several places.

For foreign visitors (who are a potential user group) toilet is likely to be a more recognised word than 'loo'.

**Comment by Hoolean** [comment]
At the moment, our usage is fragmented.

Places we use "loo":
* On the home page, in our message telling the user that their are no loos nearby
* [On the about page](https://gbptm-unity.herokuapp.com/about)
* [On the add/edit page](https://gbptm-unity.herokuapp.com/loos/574db13bdb14a11000cb4766/edit)
* In our web addresses themselves (e.g. `https://gbptm-unity.herokuapp.com/loos/574db13bdb14a11000cb4766/edit`)

Places we use "toilet":
* Basically everywhere else we refer to one (including the name of the site)

@gailramster I would be interested to hear your opinion on whether we should aim for total consistency and, if so, which term we should use.

**Comment by gailramster** [comment]
Good spot - not something I'd noticed or thought about before (other than using 'loo' occasionally in blog-style content to sound friendly). I think we should change it to toilet on the add/edit including 'toilet name', and the home page. We'll re-write the about page content anyway. The web address doesn't matter.
